playbooks/tester/integration/common/demo_tenant.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: Install packages required to create tenant/network
+  hosts: tester
+  gather_facts: no
+  sudo: yes
+  tasks:
```

Comment:
Ansible 2 uses `shade` for openstack, so it's better to use pip module and install `shade` package